src/dev/run_find_plugins_with_circular_deps.ts

```diff
@@ -42,6 +42,7 @@ const allowedList: CircularDepList = new Set([
   'src/plugins/vis_default_editor -> src/plugins/visualize',
   'src/plugins/visualizations -> src/plugins/visualize',
   'x-pack/plugins/actions -> x-pack/plugins/case',
+  'x-pack/plugins/case -> x-pack/plugins/security_solution',
```

Comment:
@mistic `security_solution` has a hard dependency on `case`, and will until the case UI is removed out of `security_solution`. I believe the dependency should be included in this list

Reply:
@stephmilovic we need to remove all the circular dependencies in order to move forward with https://github.com/elastic/kibana/issues/80508. We can add this item here but it will add up on more future work you'll need to do in order to assure we can move forward with ts project references.

Reply:
We have an issue to [fix our dependency on `lists`](https://github.com/elastic/security-team/issues/541). I'll add one for `cases` as well.

Reply:
Got it in just in time for planning :) https://github.com/elastic/security-team/issues/609